components/analytics/clickhouse/init.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+CLICKHOUSE_DB="${CLICKHOUSE_DB:-cvat}";
+CLICKHOUSE_USER="${CLICKHOUSE_USER:-user}";
+CLICKHOUSE_PASSWORD="${CLICKHOUSE_PASSWORD:-password}";
+
+cat <<EOT > /etc/clickhouse-server/users.d/user.xml
+<yandex>
+  <!-- Docs: <https://clickhouse.tech/docs/en/operations/settings/settings_users/> -->
+  <users>
+    <${CLICKHOUSE_USER}>
```

Comment:
Probably it is necessary to check that CLICKHOUSE_USER value without spaces and special symbols.